components/notebook-crd/config/default/kustomization.yaml

```diff
@@ -0,0 +1,49 @@
+# Adds namespace to all resources.
+namespace: notebook-crd-system
+
```

Comment:
this is sort of a departure from our normal practice of using a ksonnet package. I might be nice if we wait until kustomize is supported by kfctl